### ISSUE TITLE
Debian Repository / PPA

### ISSUE THREAD

**Issue by Huskynarr** [AlDanial/cloc#73]
There is actually the Version 1.60, can you update this please?

**Comment by AlDanial** [comment]
I don't own the Debian prepository entry for cloc but I'll try to find out who does.

**Comment by AlDanial** [comment]
Just an update:  I emailed the Debian cloc maintainer on April 2nd but haven't received a response yet.

**Comment by shirishag75** [comment]
Hi all, 
I did file bugs.debian.org/823949 and hoping to see if something happens. It is possible that the maintainer may be MIA (Missing in Action) but that will be known only after a month as I filed the bug today. I also saw the popcon for cloc https://qa.debian.org/popcon.php?package=cloc and in this case if the maintainer is missing, it would be transferred to Debian QA and finally dropped until/unless a new maintainer is found. 

If somebody knows packaging, they could always make an NMU https://wiki.debian.org/NonMaintainerUpload  which is possible although with the changes I shared, I fear it probably would involve a bit of repackaging. 

I do not have the requisite skillsets otherwise would have helped. Hope this helps. Thank you for your fine work.

**Comment by s7jones** [comment]
This would be incredibly useful. I haven't been able to find an up to date PPA or deb file, and there's a lot of crucial languages that are missing for us on Debian/Ubuntu-derivatives.

**Comment by f2404** [comment]
Well, you can always download the script from here and place it e.g. to `~/bin/`.

**Comment by oorestisime** [comment]
Hello, 
I ve prepared a new release (NMU) for Cloc in Debian.
It will get a review in the next days and hopefully get uploaded by the end of the week.

I ll update this as soon as it lands in unstable

**Comment by oorestisime** [comment]
So cloc latest version is in queue to be uploaded. I did a NMU. 4 days  remaining till it reaches unstable. You can track progress here https://ftp-master.debian.org/deferred.html
I ll let you close this issue if you feel like it.

**Comment by AlDanial** [comment]
Thanks for the update.

fyi, v1.68 had a problem on Arch Linux, ref https://github.com/AlDanial/cloc/issues/105, and possibly Fedora too, so I plan to release a bugfix, v1.70, on July 1.  (I'm still puzzled by how Debian/Ubuntu could have such divergent behavior from Arch/Fedora.)

**Comment by oorestisime** [comment]
If 1.70 contains that bugfix then i will possibly not go through the pain of doing a NMU again right now!
After all it doesn't affect debian/ubuntu so...

**Comment by AlDanial** [comment]
v1.70 is just released; contains the fix
